ansible.cfg

```diff
@@ -1,5 +1,5 @@
 [defaults]
-callback_whitelist=ansible.posix.profile_tasks
```

Comment:
There is a deprecation warning in the logs for this, telling us to switch to the new naming `callbacks_enabled`.